### PR TITLE
Remove {product-version}

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s):4.14+

Issue: follow-up PR for https://github.com/openshift/openshift-docs/pull/63336

Link to docs preview: https://66371--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-restricted-networks-azure-user-provisioned

QE review: QE approval is not required for this PR

Additional information:

### CODE REVIEW

**Comment by gabriel-rh** [comment]
Just beware - there is a silent build failure here:

ERROR (installing/installing_azure/installing-restricted-networks-azure-user-provisioned.adoc): "xref:../../installing/installing_azure/manually-creating-iam-azure.adoc#manually-creating-iam-azure[manually created and maintain IAM credentials]" appears to try to reference a file not included in the "openshift-enterprise" distro

**Comment by xenolinux** [comment]
@gabriel-rh It's okay to merge this PR now right?

File: installing/installing_azure/installing-restricted-networks-azure-user-provisioned.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-In {product-title} version {product-version}, you can install a cluster on Microsoft Azure by using infrastructure that you provide.
+In {product-title} version 4.14, you can install a cluster on Microsoft Azure by using infrastructure that you provide.
```
Comment (snarayan-redhat) [inline installing/installing_azure/installing-restricted-networks-azure-user-provisioned.adoc:9]: based on this fix, I understand that this feature is available only in 4.14. Not in the future versions. 
Can you confirm?

**Review by snarayan-redhat — COMMENTED** [review]
LGTM. Just one question.

File: installing/installing_azure/installing-restricted-networks-azure-user-provisioned.adoc (hunk 1 above)
Comment (xenolinux) [reply installing/installing_azure/installing-restricted-networks-azure-user-provisioned.adoc]: Addressed

**Comment by xenolinux** [comment]
This is ready to be merged; build passed cleanly

**Review by bergerhoffer — COMMENTED** [review]
LGTM for merging!

**Pull request merged**